server/api/socketService.js

```diff
@@ -35,12 +35,15 @@ const socketService = (socket) => {
     socket.emit('connected', roomId);
   });
 
+  socket.on('update-direction', updateDirection);
```

Comment:
also, use event names in that format. On official socket.io page, they don't use camel case. 
We can use lowercase words separated by a dash.

Reply:
Done